addons/html_editor/static/src/core/clipboard_plugin.js

```diff
@@ -600,6 +601,37 @@ export class ClipboardPlugin extends Plugin {
         if (!isHtmlContentSupported(ev.target)) {
             return;
         }
+        const selection = this.dependencies.selection.getEditableSelection();
+
+        const range = document.createRange();
+        range.setStart(selection.anchorNode, selection.anchorOffset);
+        range.setEnd(selection.focusNode, selection.focusOffset);
+        const selectionRect = range.getBoundingClientRect();
+
+        const caretRange = this.document.caretRangeFromPoint(ev.clientX, ev.clientY)
```

Comment:
`caretRangeFromPoint` would not work for firefox, you'll need a fallback on `caretPositionFromPoint`